doc/ref_model/chapters/chapter05.md

```diff
@@ -255,13 +289,20 @@ The configurations specified in here will be used in specifying the actual hardw
 <a name="5.4.1"></a>
 ### 5.4.1 Compute Resources
 
-
+| Reference | Feature | Description | Basic Type | Network Intensive |
+|---------------------|-----------|---------------------------|--------|--------|
+| nfvi.hw.cpu.cfg.001 | Number of CPU (Sockets) | This determines the minimum number of CPU sockets within each host | 2| 2|
+| nfvi.hw.cpu.cfg.002 | Number of Cores per CPU | This determines the number of cores needed per each CPU. | 20 | 20 |
+| nfvi.hw.cpu.cfg.003 | NUMA | NUMA support and BIOS configured to enable NUMA | N | Y |
+| nfvi.hw.cpu.cfg.004 | Simultaneous Multithreading/Hyperthreading (SMT/HT) | This allows a CPU to work multiple streams of data simultaneously | Y | Y|
```

Comment:
"This allows a CPU to work multiple streams of data simultaneously " -- CPU, data? Or "With this each CPU core supports multiple simultaneous threads of execution."

Reply:
@pgoyal01 your reviews here are not related to the purpose of the PR. If it OK with you, I will stick to the scope of the PR if possible.